src/proto/mod.rs

```diff
@@ -758,7 +835,7 @@ impl EncryptedData {
         &self,
         primary_key: &KdcPrimaryKey,
     ) -> Result<EncTicketPart, KrbError> {
-        let key_usage = 3;
+        let key_usage = 2;
```

Comment:
I assume this change is from https://www.rfc-editor.org/rfc/rfc4120#section-7.5.1 as this is encrypted with the service key?

Reply:
Yes and also in section 5.3:
```
enc-part
      This field holds the encrypted encoding of the EncTicketPart
      sequence.  It is encrypted in the key shared by Kerberos and the
      end server (the server's secret key), using a key usage value of
      2.
```